src/compiler/compiler.lisp

```diff
@@ -1427,6 +1427,11 @@
 (define-builtin objectp (x)
   (convert-to-bool `(=== (typeof ,x) "object")))
 
+;;; @VKM 30.10
```

Comment:
We can easily use git blame to get this contextual information. So unless it is really important like a historical note / TODO or something, I'd just remove it.

Reply:
fixed